src/main/kotlin/calculator/ArithmeticExpression.kt

```diff
@@ -0,0 +1,21 @@
+package calculator
+
+private const val DELIMITER = " "
+
+data class ArithmeticExpression(
+    private val expression: String
```

Comment:
```suggestion
data class ArithmeticExpression(
    expression: String
```
expression은 생성할 때만 필요한 것 같아요. 프로퍼티로 갖고 있지 않아도 될 것 같습니다 😄

Reply:
data class로 정의하다 보니 val / var 프로퍼티가 존재해야하더라구요. 😢

Reply:
앗 제가 로컬에서 class로 바꿔두고 코멘트를 남기다보니 이렇게 남았네요 😅

data class를 사용하실 거라면 해당 클래스가 갖는 데이터가 주생성자에 위치하는 것이 좀 더 자연스럽지 않을까요?
이 클래스가 어떤 데이터를 들고 있을지에 대한 정보가 주생성자에 위치하고, 식 -> 데이터로 바꾸는 과정은 부생성자나 팩토리 메서드 등에 위치하게 하는 것도 좋을 거 같아요.